classes/controller/AdminController.php

```diff
@@ -573,9 +574,18 @@ public function __construct($forceControllerName = '', $default_theme_name = 'de
             $this->can_import = true;
         }
 
+        if ($this instanceof AdminSearchConfController) {
```

Comment:
This controller is called after original controller was called and it doesn't have correct metadata so it overrides correct information